pkg/apis/config/types.go

```diff
@@ -35,6 +35,8 @@ type CoschedulingArgs struct {
 	KubeMaster string
 	// KubeConfigPath for scheduler
 	KubeConfigPath string
+	// EnableResourceCheck enables check cluster resource at pre-filer
```

Comment:
// XYZ applies additional logic in PreFilter phase to check whether
// minimum resource request of a PodGroup can be satisfied.
// It defaults to false if not set.

(XYZ to be determined)